src/playground/widget-editor/src/components/playground-renderer/component.js

```diff
@@ -1,18 +1,24 @@
 import React from "react";
 import { useSelector } from 'react-redux'
+import { getOutputPayload } from '@widget-editor/core';
 
 import RwAdapter from "@widget-editor/rw-adapter";
 import Renderer from "@widget-editor/renderer";
 
-// eslint-disable-next-line
-import widgetTest from "./test-widget";
+import providers from '../editor/mapbox-providers';
+
+import {
+  MAPSTYLES,
+  VIEWPORT
+} from '../editor/map-config';
 
 const PlaygroundRenderer = () => {
-  const widget = useSelector(state => {
-    return state.widgetEditor.widgetConfig;
+  // Returns the currently modified widget in the renderer
+  const widgetConfig = useSelector(state => {
+    return getOutputPayload(state.widgetEditor, window.WE_adapter)?.widgetConfig;
```

Comment:
Couldn't you use the `exposed-hooks` package to retrieve the adapter's instance?
```suggestion
    return getOutputPayload(state.widgetEditor, getLocalCache().adapter)?.widgetConfig;
```